src/cogs/trivia.py

```diff
@@ -112,6 +147,12 @@ def check(message) -> bool:
                     self.tally_scores(round_result)
                     await ctx.send(str(resp.author) + ' Wins!')
                     locked_channels.remove(ctx.channel.id)
+        else:
+            if ctx.channel.id not in available_channels:
+                return await ctx.send("This isn't a valid trivia channel!")
+
+            elif ctx.channel.id in locked_channels:
+                return await ctx.send("There is already a game in this channel!")
```

Comment:
Just fail silently here, the message will disrupt the game.

Reply:
Done